### ISSUE TITLE
Nuget package doesnt exists anymore or the URL is wrong on the readme

### ISSUE THREAD

**Issue by MarioGK** [AvaloniaUI/AvaloniaAutoGrid#3]

**Comment by rbwhitaker** [comment]
Yeah, I don't think it is just the URL in the readme being wrong. I've searched NuGet.org, and it doesn't seem to be there.

**Comment by guilhermeolisi** [comment]
Has this project been abandoned? the nuget package no longer exists and no updates since 2018. Unfortunately this is a very interesting control
